src/ai/backend/agent/docker/intrinsic.py

```diff
@@ -597,9 +598,29 @@ async def sysfs_impl(container_id):
                     e,
                 )
                 return None
+            async with closing_async(Docker()) as docker:
+                container = DockerContainer(docker, id=container_id)
+                data = await container.show()
+                sandbox_key = data["NetworkSettings"]["SandboxKey"]
```

Comment:
How about storing this sandbox key in `kernel_registry`?

Reply:
I created a new issue for this: #1196.